.travis.yml

```diff
@@ -46,7 +46,7 @@ before_install:
     fi
 
 install:
-  - moodle-plugin-ci install -vvv
+  - MOODLE_APP=true moodle-plugin-ci install -vvv
```

Comment:
While this is correct, maybe I'll set the variable above (`global.env`), just to be like the rest of main variables and because the dist and explained files are putting it there.

Reply:
I see, ok. NVM.